cmd/corectl/main.go

```diff
@@ -406,6 +407,19 @@ func joinCluster(client *rpc.Client, args []string) {
 	dieOnRPCError(err)
 }
 
+// evictNode evicts a Chain Core cored process from the cluster.
+// It does not modify the allowed-member list.
```

Comment:
I was thinking since allow-address and init/join are separate actions, evict and disallow-address should be separate.

They do seem like potentially semantically different things. If for example a host running cored dies, you might evict the node to reconfigure the quorum. You might replace the host and reuse the same address.